rules/crowdstrike_rules/crowdstrike_suspicious_cron.yml

```diff
@@ -0,0 +1,72 @@
+AnalysisType: rule
+Filename: crowdstrike_suspicious_cron.py
+RuleID: "Crowdstrike.SuspiciousCron"
+DisplayName: "Crowdstrike - Suspicious cron detected"
+Enabled: true
+LogTypes:
+  - Crowdstrike.FDREvent
+Tags:
+  - Crowdstrike.FDREvent
+  - Execution:Scheduled Task/Job
```

Comment:
I think this passed lint/schema validation because there is no space after the colon. I recommend finding a separator that is not colon for this text.